x/slashing/keeper.go

```diff
@@ -47,6 +47,13 @@ func (k Keeper) handleDoubleSign(ctx sdk.Context, addr crypto.Address, infractio
 		panic(fmt.Sprintf("Validator consensus-address %v not found", consAddr))
 	}
 
+	// Double sign too old
+	maxEvidenceAge := k.MaxEvidenceAge(ctx)
```

Comment:
I don't think there is a need to store this in a separate variable unless it's used elsewhere.